.github/ISSUE_TEMPLATE/tech-issue.md

```diff
@@ -0,0 +1,21 @@
+---
+name: Tech Issue
+about: Engineering-specific technical work that is not product-specific. Engineering team "owns" these issues.
+title: ''
+labels: tech
```

Comment:
Should we also add the (proposed) priority label (`P0`, `P1`, `P2`) here so that the creator doesn't forget to add one?

Reply:
i'd prefer them to add one explicitly rather than defaulting, and triage will assign otherwise, so it we won't be priority-less.